content/04.heterogeneity.md

```diff
@@ -1,17 +1,17 @@
+Unsupervised methods, in finding low-dimensional patterns in data, can ‘compress’ information from a large number of features into a smaller number of features [@doi:10.1007/978-3-540-33037-0; @doi:10.1098/rsta.2015.0202, @https://www.jmlr.org/papers/v9/vandermaaten08a.html; @https://arxiv.org/abs/1802.03426] (Figure [@fig:2]).
 A method commonly used for dimensionality reduction is principal components analysis (PCA).
-PCA identifies new features, termed _principal components_ (PCs), that are combinations of original features.
+PCA identifies higher order features, termed _principal components_ (PCs), that are combinations of original features.
 The PCs are calculated in a way that maximizes the amount of information (variance) they contain and ensures that each PC is uncorrelated with the other PCs. [@doi:10.1098/rsta.2015.0202]
-In practice, researchers often use the first few PCs to reduce the dimensionality without removing what may be important variability in the data.
-Other methods like multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) can also help identify useful patterns in the data, though t-SNE and UMAP require adjusting hyperparameters to get results that are not misleading or not reproducible. [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002]
+In practice, researchers often use the first few PCs to reduce the dimensionality without removing what may be important biological variability in the data.
+Nguyen and Holmes highlight the use of "elbow method" to select the number of appropriate dimensions. [@doi:10.1371/journal.pcbi.1006907]
+Multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) are other popular dimension reduction methods, often used for low-dimensional visualization and interpretation of data [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002]
 Testing multiple dimensionality reduction methods may be necessary to obtain a more comprehensive portrait of the data. [@doi:10.1186/s13059-020-02021-3]
-Nguyen and Holmes discuss additional important considerations for using dimensionality reduction methods such as selection criteria and interpretation of results. [@doi:10.1371/journal.pcbi.1006907]
-Beyond dimensionality reduction, other unsupervised learning approaches such as k-means or hierarchical clustering have also been used to characterize structure in genomic or imaging data. [@doi:10.1186/1471-2105-9-497; @doi:10.1109/jbhi.2013.2276766]
+Other unsupervised learning approaches such as k-means or hierarchical clustering are used to characterize structure in genomic and imaging data. [@doi:10.1186/1471-2105-9-497; @doi:10.1109/jbhi.2013.2276766]
 
```

Comment:
```suggestion
```

Since the below paragraph is now two sentences, lets just append them to the previous paragraph.